components/invitation_modal/__snapshots__/invitation_modal_members_step.test.jsx.snap

```diff
@@ -81,7 +86,7 @@ exports[`components/invitation_modal/InvitationModalMembersStep should match the
   >
     <h2>
       <FormattedMessage
-        defaultMessage="Invite People"
+        defaultMessage="Add or Invite People"
```

Comment:
While this may help existing users since they are familiar with the "Add" vs "Invite" terminology, I'm not sure there is a big difference in my mind between "Add" and "Invite" for new users. Perhaps once people are familiar with the new flow we can drop this subtext.